src/appleseed/renderer/modeling/light/directionallight.cpp

```diff
@@ -125,6 +125,7 @@ namespace
         virtual void sample(
             const ShadingContext&   shading_context,
             const Transformd&       light_transform,
+            const Vector3d&         target_point,
```

Comment:
Dummy target_point parameters, never used in any light except sun light. Third version of sample should added to light.h probably?